### ISSUE TITLE
Error occurred with status code: -11. Error message: .     

### ISSUE THREAD

**Issue by GooLey1025** [JiaoLab2021/EVG#8]
[2024-11-28 13:35:35,648] CMD: zcat /public/home/cszx_huangxh/qiujie/collabrators/gulei/biosoftwares/EVG/test/test2.sim1.fq.gz 1>/public/home/cszx_huangxh/qiujie/collabrators/gulei/biosoftwares/EVG/test/genotype/PanGenie/test2.fq && zcat /public/home/cszx_huangxh/qiujie/collabrators/gulei/biosoftwares/EVG/test/test2.sim2.fq.gz 1>>/public/home/cszx_huangxh/qiujie/collabrators/gulei/biosoftwares/EVG/test/genotype/PanGenie/test2.fq
[2024-11-28 13:35:35,696] [EVG.getsize] file exists, skip: /public/home/cszx_huangxh/qiujie/collabrators/gulei/biosoftwares/EVG/test/genotype/PanGenie/test2_genotyping.vcf
Error occurred with status code: -11. Error message: .

Follow the command:`EVG -r test.fa -v test.vcf.gz -s sample.txt --software VG-MAP VG-Giraffe GraphAligner Paragraph BayesTyper GraphTyper2 PanGenie &>log.txt &`
My PanGenie version is v3.0.1，I do not know why this happen

**Comment by GooLey1025** [comment]
[2024-11-28 14:32:09,385] CMD: PanGenie -s test2 -i /public/home/cszx_huangxh/qiujie/collabrators/gulei/biosoftwares/EVG/test/genotype/PanGenie/test2.fq -f /public/home/cszx_huangxh/qiujie/collabrators/gulei/biosoftwares/EVG/test/convert/PanGenie/out -t 10 -j 10 -o /public/home/cszx_huangxh/qiujie/collabrators/gulei/biosoftwares/EVG/test/genotype/PanGenie/test2
[2024-11-28 14:32:10,735] CMD: bayesTyperTools makeBloom -k /public/home/cszx_huangxh/qiujie/collabrators/gulei/biosoftwares/EVG/test/genotype/BayesTyper/0/test1.bam -p 10
Error occurred with status code: -11. Error message: .

**Comment by duzezhen** [comment]
Apologies for the delayed response. I have been a bit busy recently and didn’t get a chance to check the issue earlier.

Could you please try running `PanGenie` in your current environment to see if it works successfully? There is a possibility that another instance of `PanGenie` in the conda environment is causing the conflict.

**Comment by afurches** [comment]
I am having the same error. PanGenie seems to work fine in my current EVG conda environment, and there don't seem to be any competing installs on my path. I see this error in association with BayesTyper. 

The previous step (`kmc`) completes successfully and the `kmc_pre` and `kmc_suf` files are present. The .err file has no other warnings or errors. Changing the number of threads or available memory have no effect. I also tried starting over with fresh directories instead of using the `--force` option, to no avail. 

Thanks

```
CMD: bayesTyperTools makeBloom -k 2283.bam -p 8
Error occurred with status code: -11. Error message: .
```

**Comment by afurches** [comment]
In case anyone comes across this, I narrowed the BayesTyper errors down to gcc compiler incompatibilities with my HPC system. I solved the problem by building a custom Apptainer, and within it (1) installed dependencies using conda/mamba (including jellyfish, for the PanGenie manual build), (2) did a manual build of BayesTyper, (3) did a manual build of PanGenie >= v3.0, (4) did a manual build of EVG within the container. 

I'm happy to share the container with anyone interested. I'll post it in a repository when I get a chance and make sure it doesn't have glaring bugs.
